src/60_Samples_%26_Templates/Housing.html

```diff
@@ -126,51 +153,55 @@
     <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
     </head>
     <body>
-            <!-- ## Heading -->
-            <!--[amp-fit-text](https://www.ampproject.org/docs/reference/extended/amp-fit-text.html) can be used to fit the text to the available space. It works well for titles that vary not too much in length. -->
-        <h4 class="mx1">
+        <!-- ## Heading -->
+        <!--[amp-fit-text](https://www.ampproject.org/docs/reference/extended/amp-fit-text.html) can be used to fit the text to the available space. It works well for titles that vary not too much in length. -->
+        <div>
+          <h4>
             <amp-fit-text width="300" height="45" layout="responsive" max-font-size="45">
                     Cosy House in Central London
             </amp-fit-text>
-        </h4>
-        <h4 class="price-description mx1">£430,000</h4>
-        <p id="housing-description" class="m1">London SW1A 1AA</p>
+          </h4>
+          <h4 class="price-description">£430,000</h4>
+          <p id="housing-description">London SW1A 1AA</p>
```

Comment:
Can you change the `h4` into a `div`. Using h4 for styling here is not good as it's semantically wrong as the price is not a heading.

Reply:
Another nit: change `housing-description` from an `id` to a `class` for consistency.

Reply:
done